### ISSUE TITLE
[TW-616] "1.0" gets taken as "1.000000"

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#651]
_Bryce Harrington on 2012-02-09T07:12:10Z says:_ 

 Guess there's some sort of floating point number interpolation going on.

humber:~/Files/Admin$ task add "Release version 1.0"
Created task 803.
humber:~/Files/Admin$ task | grep "Release version"
803                          Release version 1.000000

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-02-09T07:12:10Z
Modified: 2014-02-09T02:06:30Z
```

**Comment by taskwarrior** [comment]
_Aikido Guy on 2012-02-09T23:41:41Z says:_ 

 Hope this bug gets fixed... it also affects outlines http://taskwarrior.org/boards/6/topics/2042

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-02-12T13:43:02Z says:_ 

 Cannot replicate.  Added bug.924.t regression tests anyway.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-02-12T14:08:39Z says:_ 

 Appears to be fixed along with #836 and #849.

**Comment by taskwarrior** [comment]
_Bryce Harrington on 2012-02-13T17:32:35Z says:_ 

 Thanks, confirmed issue can't be reproduced in latest git.
